lib/iris/tests/test_coordsystem.py

```diff
@@ -188,6 +188,24 @@ def test_as_cartopy_globe(self):
         self.assertEqual(res, expected)
 
 
+class Test_GeogCS_as_cartopy_projection(tests.IrisTest):
+    def test_as_cartopy_projection(self):
+        geogcs_args = {
+            "semi_major_axis": 6543210,
+            "semi_minor_axis": 6500000,
+            "longitude_of_prime_meridian": 30,
+        }
+        cs = GeogCS(**geogcs_args)
+        cs_projection = cs.as_cartopy_projection()
+        exps = [
+            f"a={geogcs_args['semi_major_axis']}",
+            f"b={geogcs_args['semi_minor_axis']}",
+            f"lon_0={geogcs_args['longitude_of_prime_meridian']}",
+        ]
+        for exp in exps:
+            self.assertTrue(exp in cs_projection.srs)
```

Comment:
The test below (Test_GeogCS_as_cartopy_crs) compares cartopy projections directly. Is there an advantage to checking the attributes directly here instead? I could see the below approach perhaps being less sensative to changes to cartopy.

Reply:
I've changed this as suggested, though I prefer a slightly more explicit way of entering values than the one in the test below so that it's clear I expect the same value, not just a number that's the same.